build_barback/lib/src/transformer/transformer.dart

```diff
@@ -60,12 +60,11 @@ class BuilderTransformer implements Transformer, DeclaringTransformer {
       return;
     }
 
-    // Run the build step.
-    var logger = new Logger('${input.id}');
-    Logger.root.level = Level.ALL;
-    var logSubscription = logger.onRecord.listen((LogRecord log) {
-      if (log.loggerName != logger.fullName) return;
+    hierarchicalLoggingEnabled = true;
```

Comment:
Yeah, I *think* this should be safe since
a) we're turning it on just in time (and turning it back on if someone went behind our back since the last time). Someone would have to sneak in during an async event loop and flip it to false to mess with this
b) If you're expecting the `false` case I'm pretty sure you're also fine with the behavior of the `true` case, other than possibly some worse performance